templates/seL4VMParameters.template.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+#include <arm_vm/gen_config.h>
+
+/*- set vm_address_config = configuration[me.name].get('vm_address_config') -*/
+/*- set vm_image_config = configuration[me.name].get('vm_image_config') -*/
+/*- set linux_address_config = configuration[me.name].get('linux_address_config') -*/
+/*- set linux_image_config = configuration[me.name].get('linux_image_config') -*/
+
+/*- if vm_address_config -*/
+
+unsigned long ram_base = /*? vm_address_config.get('ram_base') ?*/;
+unsigned long ram_paddr_base = /*? vm_address_config.get('ram_paddr_base') ?*/;
+unsigned long ram_size = /*? vm_address_config.get('ram_size') ?*/;
+unsigned long ram_offset = /*? vm_address_config.get('ram_offset') ?*/;
+unsigned long dtb_addr = /*? vm_address_config.get('dtb_addr') ?*/;
+unsigned long initrd_max_size = /*? vm_address_config.get('initrd_max_size') ?*/;
```

Comment:
Do we want to drop this, because it is not used anywhere? It could be added if there is a use case.

Reply:
Drop in the new vm_address_config block. It has been left in the linux_address_config block for any backwards compatibility.